schedule.py

```diff
@@ -136,15 +168,33 @@ def read_input(input_file):
                 section = Section(course_name, section_num, units, semester)
                 sections[section.name] = section
 
+    # create Time objects
+    # Time have start time, end time, list of 1/0 for weekdays, and list of conflicts with all time slots
+    times = []
+    for _, rows in time_tab.iterrows():
+        info = rows.tolist()
+        times.append(Time(start=info[5], end=info[6], weekdays=info[0:5]))
+    for t in times:
+        conflicts = {}
+        for c in times:
+            if t.conflict(c):
+                conflicts[c] = 1
+            else:
+                conflicts[c] = 0
+        t.conflicts = conflicts
```

Comment:
I don't think each `Time` object has a `.conflicts` member variable. It should be defined in the class itself, even if set to `None` or an empty dictionary/set.